### PR TITLE
[Application insight] Fix update component

### PR DESCRIPTION
**Pull request by JoyerJin**
## Description

Fixed the issue that Update-AzApplicationInsights would incorrectly remove existing tags.

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1 [hunk 1]
```diff
@@ -98,5 +118,5 @@ function setupEnv() {
 }
 function cleanupEnv() {
     # Clean resources you create for testing
-    Remove-AzResourceGroup -Name $env.resourceGroup
+    # Remove-AzResourceGroup -Name $env.resourceGroup
```
Comment (vidai-msft) [inline src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1:121]: Why was the remove command commented out?

**Comment by vidai-msft** [comment]
@JoyerJin Please leave change log here as a comment.

File: src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1 (hunk 1 above)
Comment (JoyerJin) [reply src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1:121]: I have added the reason on line 71.

Comment (vidai-msft) [reply src/ApplicationInsights/ApplicationInsights.Autorest/test/utils.ps1:121]: Can we use a new resource group name? If some one needs to re-record the test case, still he cannot delete the resource group?

**Review by vidai-msft — APPROVED** [review]
LGTM

**Pull request merged**